integration/deploy/pipeline_test.go

```diff
@@ -154,6 +165,38 @@ func TestDeployPipelineManifestInsidePipeline(t *testing.T) {
 	require.True(t, k8sErrors.IsNotFound(err))
 }
 
+// TestDeployPipelineAndConsumerEnvsFromDependency tests the following scenario:
+// - Deploying a dependency
+// - Consume dependency's variable from main pipeline
+func TestDeployPipelineAndConsumerEnvsFromDependency(t *testing.T) {
```

Comment:
This test should be in the `integration/okteto` folder. That's the only folder where we can store tests that requires a cluster with okteto installed

Reply:
resolving suggestion to addressed it in https://github.com/okteto/okteto/pull/3594

Reply:
btw, done 9d45dc7e2c0a589e4648e41982a19cc143347704